### PR TITLE
Improve wording with i18n/zh-tw.toml

### PR DESCRIPTION
**Pull request by huanlin**

### CODE REVIEW

**Review by tsaibird — APPROVED** [review]

**Review by tsaibird — APPROVED** [review]

**Review by forwardhrm-mis — APPROVED** [review]

**Review by forwardhrm-mis — APPROVED** [review]

File: i18n/zh-tw.toml [hunk 1]
```diff
@@ -46,21 +46,21 @@ other = "創建文檔議題"
 [post_create_project_issue]
 other = "創建項目議題"
 [post_posts_in]
-other = "張貼在"
+other = "張貼於"
 [post_reading_time]
-other = "分鐘閱讀"
+other = " 分鐘左右可讀完"
```
Comment (chalin) [inline i18n/zh-tw.toml:51]: Based on my research, the original prompt is better here, please revert this change or explain why your new text is better.

File: i18n/zh-tw.toml [hunk 2]
```diff
@@ -46,21 +46,21 @@ other = "創建文檔議題"
...
 [post_less_than_a_minute_read]
-other = "少於1分鐘"
+other = "1 分鐘內可讀完"
```
Comment (chalin) [inline i18n/zh-tw.toml:53]: Based on my research, the original prompt is better here, please revert this change or explain why your new text is better.

File: i18n/zh-tw.toml [hunk 3]
```diff
@@ -46,21 +46,21 @@ other = "創建文檔議題"
...
-other = "這是本節的多頁可打印視圖。"
+other = "這是本節的多頁可列印視圖。"
 [print_click_to_print]
-other = "點擊此處打印"
+other = "點擊此處列印"
 [print_show_regular]
 other = "返回此頁面的常規視圖"
 [print_entire_section]
-other = "打印整個章節"
+other = "列印整個章節"
```
Comment (chalin) [inline i18n/zh-tw.toml:63]: 👍

File: i18n/zh-tw.toml [hunk 4]
```diff
@@ -7,14 +7,14 @@ other = "上一頁"
 other = "下一頁"
 
 [ui_read_more]
-other = "更多"
+other = "閱讀全文"
```
Comment (chalin) [inline i18n/zh-tw.toml:10]: 👍

File: i18n/zh-tw.toml [hunk 5]
```diff
@@ -7,14 +7,14 @@ other = "上一頁"
...
 
 [ui_search]
-other = "站內搜索…"
+other = "站內搜尋…"
```
Comment (chalin) [inline i18n/zh-tw.toml:13]: 👍

File: i18n/zh-tw.toml [hunk 6]
```diff
@@ -7,14 +7,14 @@ other = "上一頁"
...
 
 # Used in sentences such as "Posted in News"
 [ui_in]
-other = "在"
+other = "於"
```
Comment (chalin) [inline i18n/zh-tw.toml:17]: 👍

File: i18n/zh-tw.toml [hunk 7]
```diff
@@ -30,7 +30,7 @@ other = "隱私政策"
 
 # Post (blog, articles etc.)
 [post_byline_by]
-other = "藉由"
+other = "作者："
```
Comment (chalin) [inline i18n/zh-tw.toml:33]: For consistency, I'd rather omit the punctuation here:

```suggestion
other = "作者"
```

**Review by chalin — COMMENTED** [review]
Thanks for the updates. See inline comments.

File: i18n/zh-tw.toml (hunk 2 above)
Comment (wlbksy) [reply i18n/zh-tw.toml:53]: @chalin As a matter of fact, in Chinese (no matter Simplified or Traditional), "less than a mimute" is quite a broken phrase or sentence. In my opinion, this one and the one above read much better in both Simplified and Traditional Chinese

File: i18n/zh-tw.toml (hunk 7 above)
Comment (huanlin) [reply i18n/zh-tw.toml:33]: Actuall if you removed the `:` symbol, it reads odd in Chinese. I can live with that. I'm just try to make it perfect in Chinese expression.

File: i18n/zh-tw.toml (hunk 2 above)
Comment (chalin) [reply i18n/zh-tw.toml:53]: Thank you for the feedback. Ok, let's go with your update then.

**Review by chalin — APPROVED** [review]
Thanks all for the updates, your feedback, and reviews ✨ 

I can't rebase (probably because you named your branch  PR `main`). After you rebase I'll merge.

**Comment by chalin** [comment]
Finally, I was able to an update via merging.
